### PR TITLE
Add mixed body size SMTP Simulation

### PR DESCRIPTION
**Pull request by blackheaven**

### CODE REVIEW

File: src/test/scala-2.12/org/apache/james/gatling/simulation/Injection.scala [hunk 1]
```diff
@@ -8,6 +8,7 @@ import scala.concurrent.duration._
 object Injection {
 
   private def toUserPerSec(userPerHour: Double): Double = userPerHour / 1.hour.toSeconds
+  def toUsersPerHour(userPerHour: Double): Double = userPerHour * 1.hour.toSeconds
```
Comment (chibenwa) [inline src/test/scala-2.12/org/apache/james/gatling/simulation/Injection.scala]: Isn't it

`def toUsersPerHour(userPerSecond: Double): Double = userPerSecond * 1.hour.toSeconds`

?

**Review by chibenwa — APPROVED** [review]

File: src/test/scala-2.12/org/apache/james/gatling/simulation/Injection.scala (hunk 1 above)
Comment (mbaechler) [reply src/test/scala-2.12/org/apache/james/gatling/simulation/Injection.scala]: right now it doesn't make sense, your suggestion is quite likely

Comment (blackheaven) [reply src/test/scala-2.12/org/apache/james/gatling/simulation/Injection.scala]: likely to...?

**Comment by blackheaven** [comment]
Rebase: commit typo

**Review by remk — APPROVED** [review]

**Review by chibenwa — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Review by chibenwa — APPROVED** [review]

File: src/test/scala-2.12/org/apache/james/gatling/simulation/SimulationOnMailCorpus.scala [hunk 2]
```diff
@@ -29,7 +29,7 @@ trait SimulationOnMailCorpus {
 
   protected def injectUsersInScenario(scenario: ScenarioBuilder, nbUsers: UsersDensity = UsersPerHour(50000)) = {
     scenario
-      .inject(nbUsers.constantUserPerHour during 1.hour)
+      .inject(nbUsers.constantUserPerHour during Configuration.ScenarioDuration)
```
Comment (remk) [inline src/test/scala-2.12/org/apache/james/gatling/simulation/SimulationOnMailCorpus.scala]: s/constantUserPerHour/constantUserPerScenarioDuration ?

**Review by remk — APPROVED** [review]

**Comment by blackheaven** [comment]
Merged

**Pull request closed without merging**